go.mod

```diff
@@ -1,10 +1,13 @@
 module github.com/srl-labs/containerlab
 
-go 1.21
+go 1.22
+
+toolchain go1.22.2
```

Comment:
@adrianosela it looks like `github.com/borderzero/border0-go` requires go1.22.
Is it mandatory in your case?
In containerlab we use `goN-1` version to allow for broader adoption.

Maybe go1.22 is not really needed by border0-go and you can declare go1.21

Reply:
Downgraded `github.com/borderzero/border0-go` to `go1.21` 😄